<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDevCpp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="SrcTab">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionToggleComment"/>
    <addaction name="separator"/>
    <addaction name="actionCopyRow"/>
    <addaction name="actionDelRow"/>
    <addaction name="separator"/>
    <addaction name="actionMoveRowUp"/>
    <addaction name="actionMoveRowDown"/>
    <addaction name="separator"/>
    <addaction name="actionIndent"/>
    <addaction name="actionUnindent"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>搜索</string>
    </property>
    <addaction name="actionFindReplace"/>
   </widget>
   <widget class="QMenu" name="menuVisual">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionCompileInfoDock"/>
    <addaction name="actionDebugToolDock"/>
    <addaction name="actionStatusBar"/>
   </widget>
   <widget class="QMenu" name="menu_R">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionRun"/>
    <addaction name="actionCompileRun"/>
    <addaction name="actionClean"/>
    <addaction name="separator"/>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAboutQDevCpp"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionCompileConfig"/>
    <addaction name="actionEnvironmentConfig"/>
    <addaction name="actionEditorConfig"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSearch"/>
   <addaction name="menuVisual"/>
   <addaction name="menu_R"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QDockWidget" name="dockDebug">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockInfo">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="compileInfo">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>编译信息</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="compileResult">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>文件</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>行</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>列</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>级别</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>信息</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>日志</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="Log"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="iconText">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="toolTip">
    <string>打开项目或文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>另存为...</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>全部关闭</string>
   </property>
   <property name="toolTip">
    <string>全部关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAboutQDevCpp">
   <property name="text">
    <string>关于QDevCpp</string>
   </property>
  </action>
  <action name="actionAboutDevCpp">
   <property name="text">
    <string>关于DevCpp</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>编译</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionCompileRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>编译运行</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionRunParam">
   <property name="text">
    <string>运行参数</string>
   </property>
  </action>
  <action name="actionSyntaxCheck">
   <property name="text">
    <string>语法检查</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>清理</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>调试</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>停止执行</string>
   </property>
  </action>
  <action name="actionCompileInfoDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>编译信息</string>
   </property>
  </action>
  <action name="actionDebugToolDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>调试工具</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="actionCompileConfig">
   <property name="text">
    <string>编译选项</string>
   </property>
  </action>
  <action name="actionAboutGCC">
   <property name="text">
    <string>关于GCC</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFindReplace">
   <property name="text">
    <string>搜索替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionEnvironmentConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>环境选项</string>
   </property>
  </action>
  <action name="actionEditorConfig">
   <property name="text">
    <string>编辑器选项</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionToggleComment">
   <property name="text">
    <string>设置注释</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="actionCopyRow">
   <property name="text">
    <string>复制行</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDelRow">
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionMoveRowUp">
   <property name="text">
    <string>上移选择行</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="actionMoveRowDown">
   <property name="text">
    <string>下移选择行</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Down</string>
   </property>
  </action>
  <action name="actionIndent">
   <property name="text">
    <string>缩进</string>
   </property>
   <property name="shortcut">
    <string>Tab</string>
   </property>
  </action>
  <action name="actionUnindent">
   <property name="text">
    <string>取消缩进</string>
   </property>
   <property name="shortcut">
    <string>Shift+Tab</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
